lib/mumukit/nuntius/publisher.rb

```diff
@@ -3,9 +3,9 @@ module Mumukit::Nuntius::Publisher
   class << self
 
     def publish(exchange_name, data, opts={})
-      connection, channel, exchange = Mumukit::Nuntius::Connection.start(exchange_name)
```

Comment:
Perhaps we should rename this class to `Channel`

Reply:
I'm not sure if we should rename the class Channel, or just rename the method start to `start_channel`. The method config still refers to the connection